lib/partition_healing/discover_provider_healer.js

```diff
@@ -63,6 +63,7 @@ require('util').inherits(DiscoverProviderHealer, Healer);
  */
 DiscoverProviderHealer.prototype.heal = function heal(callback) {
     var self = this;
+    self.ringpop.stat('increment', 'heal.triggered.discover_provider');
```

Comment:
Good point. I wanted to discuss this with @CorgiMan.
Since most partition healing code is already prepared for multiple strategies, I thought it made more sense to increment a more specific counter. @CorgiMan do you agree?